packages/ckeditor5-find-and-replace/src/ui/findandreplaceformview.js

```diff
@@ -47,190 +47,133 @@ export default class FindAndReplaceFormView extends View {
-			this.fire( 'replace', { searchText: this.searchText, replaceText: this.replaceText } );
-		} );
-
-		/**
-		 * The replace all button view.
-		 *
-		 * @member {module:ui/button/buttonview~ButtonView}
+		 * @readonly
+		 * @observable
+		 * @member {Boolean} #matchCase
 		 */
-		this.replaceAllButtonView = this._createButton( t( 'Replace all' ), 'ck-button-replaceall' );
-		this.replaceAllButtonView.on( 'execute', () => {
-			this.fire( 'replaceAll', { searchText: this.searchText, replaceText: this.replaceText } );
-		} );
-
-		/**
-		 * The match case checkbox view.
-		 *
-		 * @member {module:find-and-replace/ui/checkboxview~CheckboxView}
-		*/
-		this.matchCaseView = this._createCheckbox( t( 'Match case' ) );
+		this.set( 'matchCase', false );
 
 		/**
-		 * The whole words only checkbox view.
-		 *
-		 * @member {module:find-and-replace/ui/checkboxview~CheckboxView}
-		*/
-		this.matchWholeWordsView = this._createCheckbox( t( 'Whole words only' ) );
-
-		/**
-		 * The replace input view.
-		 *
-		 * @member {module:ui/labeledfield/labeledfieldview~LabeledFieldView}
-		 */
-		this.replaceInputView = this._createInputField( t( 'Replace with…' ) );
+		  * Indicates if the matchWholeWords checkbox has been checked.
+		  *
+		  * @readonly
+		  * @observable
+		  * @member {Boolean} #matchWholeWords
+		  */
+		this.set( 'matchWholeWords', false );
 
 		/**
-		 * Stores gathered views related to find functionality of the feature.
+		 * Stores the number of matched search results.
 		 *
-		 * @member {module:ui/view~View}
+		 * @readonly
+		 * @observable
+		 * @member {Number} #matchCount
 		 */
-		this.findView = this._createFindView();
+		this.set( 'matchCount', null );
 
 		/**
-		 * Stores gathered views related to replace functionality of the feature.
+		 * The offset of currently highlighted search result in {@link #matchCount matched results}.
 		 *
-		 * @member {module:ui/view~View}
+		 * @readonly
+		 * @observable
+		 * @member {Number|null} #highlightOffset
 		 */
-		this.replaceView = this._createReplaceView();
+		this.set( 'highlightOffset', null );
 
 		/**
-		 * Tracks information about the DOM focus in the form.
+		 * Whether the search results counter should be visible.
 		 *
+		 * @private
 		 * @readonly
-		 * @member {module:utils/focustracker~FocusTracker}
+		 * @observable
+		 * @member {Boolean} #isCounterHidden
 		 */
-		this.focusTracker = new FocusTracker();
+		this.set( 'isCounterHidden', true );
 
 		/**
-		 * An instance of the {@link module:utils/keystrokehandler~KeystrokeHandler}.
+		 * Assigning form elements views.
 		 *
+		 * @private
 		 * @readonly
-		 * @member {module:utils/keystrokehandler~KeystrokeHandler}
+		 * @observable
+	 	 * @param {Function} t Used to translate message to the uiLanguage.
 		 */
-		this.keystrokes = new KeystrokeHandler();
+		this._assignViews( t );
 
 		/**
-		 * A collection of views that can be focused in the form.
+	 	 * Initialization of focusables elements of the form.
```

Comment:
I believe I'm seeing a space introduced inbetween tabs which means a mixed tabs and spaces.